src/accessibility/AccessibilityMixins.d.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace PixiMixins
+{
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface Container extends Partial<import('./accessibilityTarget').AccessibleTarget>
+    {
+
+    }
```

Comment:
```suggestion
    }
    
    interface RendererSystems
    {
        accessibility: import('./AccessibilitySystem').AccessibilitySystem;
    }
```

This should add `renderer.accessibility` to the types

Reply:
nice, thanks